nilearn/plotting/img_plotting.py

```diff
@@ -117,6 +117,7 @@ def _plot_img_with_bg(img, bg_img=None, cut_coords=None,
                       display_factory=get_slicer,
                       cbar_vmin=None, cbar_vmax=None,
                       brain_color=(0.5, 0.5, 0.5),
+                      decimals=False,
```

Comment:
should `decimals` be exposed and documented in other functions e.g. plot_stat_map, plot_anat etc?

Reply:
WIll update on the next PR. Thank you for your comment!